### ISSUE TITLE
Display year in the duration card on competition page

### ISSUE THREAD

**Issue by rorro** [wise-old-man/wise-old-man#1524]
On the competition page itself, there is no way of knowing what year the competition ran. Showing the start and end years in the duration card would be nice.
![image](https://github.com/wise-old-man/wise-old-man/assets/5983417/064b987a-a7ec-4bdf-ae6a-786d70f40c86)

**Comment by godspeed-03** [comment]
Hii @rorro 
I noticed that the competition page currently doesn't display the years the competition ran. Including the start and end years is crucial as it provides context and relevance, helping viewers understand the timeframe of the competition.

I believe this enhancement would greatly benefit our viewers, and I would like to work on implementing it :smile:

**Comment by godspeed-03** [comment]
![image](https://github.com/wise-old-man/wise-old-man/assets/119731754/4152c314-250c-418d-bfe9-e20d2c58d3b5)
This is the example of the enhanced UI :eyes:

**Comment by rorro** [comment]
Yeah, feel free to work on it. In the example screenshot the card is expanded down? How do the other cards to the right scale with this? If you have a working version, feel free to open a pr for it and we can discuss it further in there.

**Comment by godspeed-03** [comment]
Hii @rorro, I have raised a pull request #1526 . KIndly look and suggest if need more changes.

**Comment by nic-w** [comment]
Possible solution without adjusting height

![Image](https://github.com/user-attachments/assets/27edb69c-8254-42a8-9f18-a256248b6201)
